worlds/stardew_valley/content/__init__.py

```diff
@@ -122,3 +123,22 @@ def choose_skill_progression(skill_progression_option: options.SkillProgression)
         raise ValueError(f"No skill progression feature mapped to {str(skill_progression_option.value)}")
 
     return skill_progression_feature
+
+
+def choose_tool_progression(tool_option: options.ToolProgression, skill_option: options.SkillProgression) -> tool_progression.ToolProgressionFeature:
+    if (tool_option == options.ToolProgression.option_vanilla
```

Comment:
Why use Ors? Aren't these usable as flags instead?

Reply:
Not that simple, since `option_vanilla` is `0b000`. I can't just use a bitwise and. Checking if it's not progressive with a bitwise and would IMO be more fragile since it would need to be revisited when a new bit will be added to the flags.

Reply:
Should we make an utility method, inside the ToolProgression option class, that is like self.is_vanilla() and self.is_progressive()

Reply:
That might be a good idea in general, but it will not change anything here